shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -71,7 +71,7 @@
  * FlutterEngine(context); flutterEngine .getDartExecutor()
  * .executeDartEntrypoint(DartEntrypoint.createDefault());
  *
- * <p>// Cache the pre-warmed FlutterEngine in the FlutterEngineCache.
+ * <p><p>// Cache the pre-warmed FlutterEngine in the FlutterEngineCache.
```

Comment:
nit: I think this implementation comment is lost

Reply:
Yeah, looks like the formatter already jumbled this. I just tried to fix the spacing, but I'm not sure how I'll get it by the linter.

Reply:
Yeah, looks like the formatter already jumbled this. I just tried to fix the spacing, but I'm not sure how I'll get it by the linter.

Reply:
Ah, I missed that this was supposed to be a code sample, sorry. I thought this was an implementation comment meant for the actual code of the class. What you want here are actually `<pre>` tags, not paragraph (weird HTML convention). I'll put in a follow up CL to find and fix the instances of code eaten like this.

Reply:
It looks like the linter wants to keep adding more `<p>` tags here. Really, we don't want any. Do you know if `<pre>` is supported in a way that would help here?

Reply:
Looks like we both had the same idea. Do you know if I should add a `<pre>` on the line above the whole block and a `</pre>` on the lien after the whole block?

Reply:
As I understand it the "right" way to do this is `<pre>{@code }</pre>`. https://stackoverflow.com/a/52002609/3968079 has a good explanation. I put up flutter/engine#16565 to try and clean this up since this was a regression from the formatting change.